.github/workflows/Test-Rock.yaml

```diff
@@ -245,6 +245,12 @@ jobs:
           repository: canonical/oci-factory
           ref: ${{ needs.configure-tests.outputs.oci-factory-ref }}
           fetch-depth: 1
+      
+      - name: Checkout Rock Repo
+        uses: actions/checkout@v4
+        if: github.repository != 'canonical/oci-factory'
+        with:
+          path: rock-repo
```

Comment:
this better be an env var since you're then using this path again in another step below